automatic/wget/README.md

```diff
@@ -17,3 +17,7 @@ GNU Wget has many features to make retrieving large files or mirroring entire we
 * Unattended / background operation
 * Uses local file timestamps to determine whether documents need to be re-downloaded when mirroring
 * GNU Wget is distributed under the GNU General Public License.
+
+## Notes
+
+**If the package is out of date please check [Version History](#versionhistory) for the latest submitted version. If you have a question, please ask it in [Chocolatey Community Package Discussions](https://github.com/chocolatey-community/chocolatey-packages/discussions) or raise an issue on the [Chocolatey Community Packages Repository](https://github.com/chocolatey-community/chocolatey-packages/issues) if you have problems with the package. Disqus comments will generally not be responded to.**
```

Comment:
Just a note for the future:

We might need to come to an agreement in the future whether we should have all notes be in a list or not, even if there is just 1 item.

In the past, we always had notes inside a list for packages.

Not too fuzzed about it, for now, just something we'll need to think about for consistency.

Reply:
I think a list is the way to go.